### PR TITLE
Add HandshakeRequest packet

### PR DESCRIPTION
**Pull request by huffman**
These were pending changes from a week ago that Clement needs for testing. It should fix a UDT issue.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: c5c35b7e89d8631fa0441b6d859d2e040ed2065b available at: https://gist.github.com/ecf1432d48060dbd946f<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7398.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7398.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: b7678f9818f5cf6647f2e8c145e1c6450d1bdd98 available at: https://gist.github.com/ecf1432d48060dbd946f<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7398.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7398.dmg

File: libraries/networking/src/udt/Connection.cpp [hunk 1]
```diff
@@ -408,7 +408,15 @@ SequenceNumber Connection::nextACK() const {
 bool Connection::processReceivedSequenceNumber(SequenceNumber sequenceNumber, int packetSize, int payloadSize) {
     
     if (!_hasReceivedHandshake) {
-        // refuse to process any packets until we've received the handshake
+        // Refuse to process any packets until we've received the handshake
+        // Send handshake request to re-request a handshake
+        static auto handshakeRequestPacket = ControlPacket::create(ControlPacket::HandshakeRequest, 0);
```
Comment (birarda) [inline libraries/networking/src/udt/Connection.cpp]: we should fix this before it causes a crash!

**Comment by hifi-gustavo** [comment]
Build console output for commit: c515355cfff89f27c43b9e4bb9640c15f3c16049 available at: https://gist.github.com/ecf1432d48060dbd946f<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7398.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7398.dmg

**Pull request merged**